views/home_massAddTable.mako

```diff
@@ -38,7 +38,7 @@
         <td align="center">
             <select name="indexer">
                 % for curIndexer in indexerApi().indexers.items():
-                    <option value="${curIndexer[0]}" ${('selected="selected"', '')[curIndexer[0] == indexer] or curIndexer[0] == app.INDEXER_DEFAULT}>${curIndexer[1]}</option>
+                    <option value="${curIndexer[0]}" ${('', 'selected="selected"')[curIndexer[0] == indexer] or curIndexer[0] == app.INDEXER_DEFAULT}>${curIndexer[1]}</option>
```

Comment:
As mentioned in IRC this should be written as `x if condition else y` instead of `(y, x)[condition]`.  It's clearer and less error prone.